python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -524,7 +524,7 @@ def fitPtc(self, dataset):
         if dataset.ptcFitType:
             ptcFitType = dataset.ptcFitType
         else:
-            raise RuntimeError(f"ptcFitType is None of empty in PTC dataset.")
+            raise RuntimeError("ptcFitType is None of empty in PTC dataset.")
```

Comment:
Thanks for fixing this type. Can you fix the other typo ("None or empty" instead of "None of empty") before you merge?

Reply:
I think this was resolved on another ticket.  I fixed this on my branch to avoid a linter error, and it looks like someone else pushed first.